elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -92,3 +96,13 @@ No symbolic expressions (those that involve fractions, square roots, variables,
 {{#shortformat}}
 {{#relabs}}(rtol={{rtol}}, atol={{atol}}){{/relabs}}{{#sigfig}}({{digits}} significant figures){{/sigfig}}{{#decdig}}({{digits}} digits after decimal){{/decdig}}
 {{/shortformat}}
+
+{{#format_error}}
+    <p> Invalid format: one or more entries of the submitted answer could not be interpreted as a decimal {{#complex}}or complex{{/complex}} number.  No symbolic input or expressions are allowed.</p>
+    <p> Example <i>valid</i> inputs: <code class="user-output">5</code> <code class="user-output">-17</code> <code class="user-output">4.25e7</code> {{#complex}}<code class="user-output">3.6i</code>{{/complex}} </p>
```

Comment:
Can we add some more space between the examples to make sure they are clearly distinct as separate items?